app/iht/views/application/asset/household/household_overview.scala.html

```diff
@@ -55,31 +55,31 @@
                     generic_overview_table_heading_item(id = AssetsHouseholdOwnID,
                         questionText = Messages("iht.estateReport.assets.householdAndPersonalItemsOwnedByDeceased.title",deceasedName),
                         questionScreenReaderText = Messages("iht.estateReport.assets.householdAndPersonalItemsOwnedByDeceased.title",
-                                                            Messages("iht.the.deceased")),
+                            deceasedName),
                         link = Some(iht.controllers.application.assets.household.routes.HouseholdDeceasedOwnController.onPageLoad),
                         linkScreenReader = getLinkScreenReader(
                             household.flatMap(_.isOwned.map(identity)),
-                            Messages("page.iht.application.assets.household.overview.deceased.isOwned.screenreader.link.change"),
-                            Messages("page.iht.application.assets.household.overview.deceased.isOwned.screenreader.link.noValue")),
+                            Messages("page.iht.application.assets.household.overview.deceased.isOwned.screenreader.link.change", deceasedName),
```

Comment:
The previous text was "Give an answer for household and personal items held in the deceased’s name" and I can't see anything on the criteria about changing the wording, other than switching deceased for the name